Tests/ShortcutFoundationTests/CollectionExtensionTests.swift

```diff
@@ -10,7 +10,7 @@ import XCTest
 
 final class CollectionExtensionTests: XCTestCase {
     
-    func testSuccessfulSafeArrayIndexing() {
+    func testSafeArrayIndexingSuccess() {
```

Comment:
Could you just refactor to use our naming conventions for naming test cases?

test_safe_array_indexing_success

Reply:
@gabrielPeart, sure thing! Fixed in ae3a577

Out of curiosity, what's the reason for this naming convention?